net/ddns-scripts/files/usr/lib/ddns/dynamic_dns_updater.sh

```diff
@@ -183,6 +183,7 @@ ERR_LAST=$?	# save return code - equal 0 if SECTION_ID found
 [ -z "$retry_count" ]	  && retry_count=0	# endless retry
 [ -z "$use_syslog" ]      && use_syslog=2	# syslog "Notice"
 [ -z "$use_https" ]       && use_https=0	# not use https
+[ -z "$user_agent" ]      && user_agent=''	# user agent for wget
```

Comment:
The way I see it, we don't need a default here.
Because if this variable is not set, then no agent string is set for a wget and it uses the default value. Whatever that is!